### PR TITLE
Add Material-UI

### PR DESCRIPTION
**Pull request by nadunindunil**
https://github.com/nadunindunil/awesome-material-ui

Material-UI is a successful react component library and it is one of the famous(more than 50k stars) react libraries.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [x] - **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [x] - Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- [x] - Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- [x] - **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- ✅ `Mobile operating system for Apple phones and tablets.`
- ✅ `Prototyping interactive UI designs.`
- ❌ `Resources and tools for iOS development.`
- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
- [x] 	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- [x] - The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
- ✅ `awesome-swift`
- ✅ `awesome-web-typography`
- ❌ `awesome-Swift`
- ❌ `AwesomeWebTypography`
- [x] - The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
- ✅ `# Awesome Swift`
- ✅ `# Awesome Web Typography`
- ❌ `# awesome-swift`
- ❌ `# AwesomeSwift`
- [x] - Non-generated Markdown file in a GitHub repo.
- [x] - The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [x] - Not a duplicate. Please search for existing submissions.
- [x] - Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] - Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] - Includes a project logo/illustration whenever possible.
- [x] 	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
- [x] 	- The image should link to the project website or any relevant website.
- [x] 	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- [x] - Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] - Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
- [x]  - Should be placed on the right side of the readme heading.
- [x] 	- Can be placed centered if the list has a centered graphics header.
- [x] 	- Should link back to this list.
- [x] - Has a Table of Contents section.
- [x] 	- Should be named `Contents`, not `Table of Contents`.
- [x] 	- Should be the first section in the list.
- [x] 	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- [x] - Has an appropriate license.
- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=master&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
- [x]  - A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org).
- [x] 	- Place a file named `license` or `LICENSE` in the repo root with the license text.
- [x] 	- **Do not** add the license name or text to the readme. GitHub already shows the license name at the top of the repo.
- [x] 	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- [x] - Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
- [x] 	- The file should be named `contributing.md`. Casing is up to you.
- [x] - Has consistent formatting and proper spelling/grammar.
- [x] 	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
- [x] 	- The description starts with an uppercase character and ends with a period.
- [x] 	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] - Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] - Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sergey-pimenov** [comment]
Hi! You must add some github topics to your repository: "awesome" and "awesome-list". Also you need to remove "Contributing" title from table of content. You can remove the "Contributing" section from readme at all (but this is not necessary). Just keep contributing.md file at repo.

After you solve these problems use [awesome-lint](https://github.com/sindresorhus/awesome-lint) to see other issues. I also strongly advise you to read  very carefully [pull_request_template.md](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md).

**Comment by nadunindunil** [comment]
@sergey-pimenov thank you very much for the instructions 
I am working on changes and I will make them as list points.
I will update and let you know these when I am finished (with `unicorn` keyword) 👍

**Comment by nadunindunil** [comment]
unicorn 🦄

**Comment by mgramin** [comment]
awesome-lint still detects several errors:

```
  ×   1:1  Missing Awesome badge after the main heading       awesome/badge
  ×   3:1  Remove empty section: "Awesome Material-UI "       empty-sections
  ×   3:1  Don’t use multiple top level headings (3:1)        no-multiple-toplevel-headings
  ×   3:3  Remove trailing slash (https://material-ui.com)    trailing-slash
  ×   5:1  Remove trailing slash (https://material-ui.com)    trailing-slash
  ×  22:3  Remove trailing slash (https://material-ui.com)    trailing-slash
  ×  76:3  Remove trailing slash (https://bethesda.net)       trailing-slash
  ×  77:3  Remove trailing slash (https://neotracker.io)      trailing-slash
  ×  78:3  Remove trailing slash (https://www.tagspaces.org)  trailing-slash
  ×  79:3  Remove trailing slash (https://builderbook.org)    trailing-slash

  10 errors
```

**Comment by mgramin** [comment]
You should review the guidelines about license again

**Comment by angrykoala** [comment]
Some of the items under `Related Projects` seem to be components, in particular material-auto-rotating-carousel and material-ui-image which seem to belong to a set of components (Wertarbyte Material-UI Components). I'm not sure about the difference between these sections (Related Projects and Components) and I feel it may lead to confusion to some people.

**Comment by nadunindunil** [comment]
I went through the awesome-lint again for the verification

**Comment by nadunindunil** [comment]
unicorn 🦄

File: readme.md [hunk 1]
```diff
@@ -294,6 +293,7 @@
 - [Tailwind CSS](https://github.com/aniftyco/awesome-tailwindcss#readme) - Utility-first CSS framework for rapid UI development.
 - [Seed](https://github.com/seed-rs/awesome-seed-rs#readme) - Rust framework for creating web apps running in WebAssembly.
 - [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Techniques to ensure certain performance metrics for a website.
+- [Material-UI](https://github.com/nadunindunil/awesome-material-ui#readme) - Material Design react components for faster and easier web development.
```
Comment (GypsyTheDj) [inline readme.md]: I like this one 😊

**Comment by nadunindunil** [comment]
@sindresorhus could you please take a look at this one

File: readme.md [hunk 2]
```diff
@@ -299,6 +299,7 @@
 - [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Techniques to ensure certain performance metrics for a website.
 - [Web Animation](https://github.com/sergey-pimenov/awesome-web-animation#readme) - Animations in the browser with JavaScript, CSS, SVG, etc.
 - [Yew](https://github.com/jetli/awesome-yew#readme) - Rust framework inspired by Elm and React for creating multi-threaded frontend web apps with WebAssembly.
+- [Material-UI](https://github.com/nadunindunil/awesome-material-ui#readme) - Material Design react components for faster and easier web development.
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- [Material-UI](https://github.com/nadunindunil/awesome-material-ui#readme) - Material Design React components for faster and easier web development.
```

Comment (sindresorhus) [reply readme.md]: This applies to your readme too.

**Comment by sindresorhus** [comment]
> The link and description are **separated by a dash**.

---

> Entries have a description

---

> These are tools and utilities to help build apps using Material-UI Design

All sentences should end in a `.`

---

It would look nicer if the logo was above the header title. I would also center the title.

---

> Consistent and correct naming. 

Applies to the repo description.

**Comment by nadunindunil** [comment]
Thank you very much for pointing out my mistakes. I am going to change the requested.

- [ ] The link and description are separated by a dash 
- [ ] Entries have a description
- [ ] All sentences should end in a "."
- [ ] It would look nicer if the logo was above the header title. I would also center the title.
- [ ] Consistent and correct naming. (Applies to the repo description.)

**Comment by ltchris** [comment]
The repo is using a MIT license, which is not accepted per the guidelines:
> A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and Unlicense.

**Comment by nadunindunil** [comment]
Hi @sindresorhus, I have completed the tasks you requested. However, I couldn't center the Material-UI title since it requires doesn't support native markup. (I moved logo above the title.)
Could you please take a look at this PR!
Thank you.
🦄

**Comment by sindresorhus** [comment]
> However, I couldn't center the Material-UI title since it requires doesn't support native markup.

This is totally possible. In fact, the readme in this repo does it.

**Comment by nadunindunil** [comment]
hi @sindresorhus ,
I have centered the Title as you requested.

Thank you.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1241624727396814848

**Pull request merged**